.circleci/config.yml

```diff
@@ -265,7 +265,7 @@ defaults:
   - base_archlinux_large: &base_archlinux_large
       docker:
         - image: archlinux:base
-      resource_class: large
+      resource_class: macos.m1.large.gen1
```

Comment:
This is a unrelated change. Please do it in a different PR. See my comment here: https://github.com/ethereum/solidity/pull/14159#issuecomment-1523711323

Please, read our contributing guides before open many PRs: https://github.com/ethereum/solidity/pull/14159#issuecomment-1523437136